README.md

```diff
@@ -68,9 +68,32 @@ Same Steps 1-7 above<br/>
 10. Choose "build > build Zelda3" in the menu to create zelda3.exe in the "/bin/release" subfolder<br/>
 11. Configure with zelda3.ini in the main dir<br/>
 
-## Compiling on Linux/MacOS
+## Installing libraries on Linux/MacOS
+1. Open a terminal
+2. Install pip if not already installed
+```sh
+curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python3 get-pip.py
+```
+3. Install `pillow` and `pyyaml` using pip
+```sh
+python3 -m pip install pillow pyyaml

```

Comment:
This was replaced with the requirements.xt

https://github.com/snesrev/zelda3/commit/37b322c5951e46bf82f59b77ab52678b1a7a7022